tests/unittest/_torch/visual_gen/multi_gpu/test_wan_ring.py

```
@@ -0,0 +1,322 @@
+    """WAN ring CP (size 4) output matches single-GPU FA4 reference.
+
+    Both models use FA4 as the inner attention backend; ring wraps it with
+    KV streaming and online LSE merge. The WAN forward all-gathers the
+    sequence at the end, so each rank can compare to the single-GPU reference.
+    """
+    from tensorrt_llm._torch.visual_gen.models.wan.transformer_wan import WanTransformer3DModel
+
+    device = torch.device(f"cuda:{rank}")
+    dtype = torch.bfloat16
+
+    B, C, T, H, W = _VIDEO_SHAPE
```

Ruff (0.15.12)

[warning] Unpacked variable `C` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)